canary/producer-c/canary/KvsProducerSampleCloudwatch.cpp

```diff
@@ -260,7 +260,10 @@ INT32 main(INT32 argc, CHAR* argv[])
         }
 
         // adjust members of pDeviceInfo here if needed
-        pDeviceInfo->clientInfo.loggerLogLevel = LOG_LEVEL_DEBUG;
+        logLevel = getenv(DEBUG_LOG_LEVEL_ENV_VAR);
+        if (logLevel != NULL) {
+            STRTOUI32(logLevel, NULL, 10, &pDeviceInfo->clientInfo.loggerLogLevel);
```

Comment:
Need to assume some default value if this fails. Something like

if (STATUS_FAILED(STRTOUI(*)) {
    pDeviceInfo->clientInfo.loggerLogLevel = DEFAULT_BLAH_BLAH_LOG_LEVEL;
}

Reply:
Default is internally set to WARN.

Reply:
Actually, now that I think about it, I can set it to LOG_LEVEL_DEBUG by default and set to provided value if env is set.

Reply:
sounds good. I am going to approve it